tests/ga/test_multi_config_extension.py

```diff
@@ -738,7 +738,7 @@ def __assert_env_variables(handler_name, handler_version="1.0.0", seq_no="1", ex
                     self.assertFalse(any(env_var in commands['data'] for env_var in not_expected), "Unwanted env variable found")
 
         def mock_popen(cmd, *_, **kwargs):
-            if 'env' in kwargs:
+            if 'env' in kwargs and ExtCommandEnvVariable.ExtensionVersion in kwargs['env']:
```

Comment:
This cgroupsapi mocking all other popen calls which breaking for non-extension commands. So, I added this check to call extension emulator on extension operations.

Reply:
let's add this (with a little more detail) as a comment in the code itself

Reply:
addressed